app/controllers/socure_webhook_controller.rb

```diff
@@ -4,6 +4,17 @@ class SocureWebhookController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def create
-    render json: { message: 'Got here.' }
+    if token_valid?
+      render json: { message: 'Secret token is valid.' }
+    else
+      render status: :unauthorized, json: { message: 'Invalid secret token.' }
+    end
+  end
+
+  private
+
+  def token_valid?
+    authorization_header = request.headers['Authorization']&.split&.last
+    authorization_header == IdentityConfig.store.socure_webhook_secret_key
```

Comment:
```suggestion
    ActiveSupport::SecurityUtils.secure_compare(
      authorization_header,
      IdentityConfig.store.socure_webhook_secret_key,
    )

```

Reply:
Should we make the config an array so we can smoothly support a transition between an old & new key?

Reply:
@theabrad ... i think @zachmargolis suggestion is worth consideration ... if not, with the current implementation, how do you suggest we rotate keys?

Reply:
i think @zachmargolis makes a great point ☝🏿  and this suggestion worth consideration ☝🏿